kodilla-exception/src/main/java/com/kodilla/exception/test/SearchEngine.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SearchEngine {
+
+    public void findFlight(Flight flight) throws RouteNotFoundException {
+
+    Map<String, Boolean> airportList = new HashMap<String,Boolean>();
```

Comment:
Samo używanie `List` w nazwie zmiennej jest złe, ale nazywanie mapy listą to już przegięcie ;)